packages/components/src/components/post-megadropdown/post-megadropdown.tsx

```diff
@@ -214,6 +244,27 @@ export class PostMegadropdown {
     }
   }
 
+  private handleAriaCurrentChange(mutations: MutationRecord[]) {
+    if (!mutations.length) return;
+
+    const el = mutations[0].target as HTMLElement;
```

Comment:
If you receive two mutations, one for a child where the `aria-attribute` was remove and one for a child where it was added, your code will only react to the first one therefore removing the `.selected` class while it shouldn't.
Please use `mutations.some(...)` to check if the trigger should be activated or deactivated.